projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -292,3 +292,28 @@ function wpcom_site_menu_handle_dismiss_notice() {
 	wp_die();
 }
 add_action( 'wp_ajax_dismiss_wpcom_site_menu_intro_notice', 'wpcom_site_menu_handle_dismiss_notice' );
+
+/**
+ * Ensures customizer menu and adminbar items are not visible on a block theme for atomic sites.
+ */
+function hide_customizer_menu_on_block_theme() {
+	$is_wpcom = ( defined( 'IS_WPCOM' ) && IS_WPCOM );
+	if ( ! $is_wpcom && wp_is_block_theme() ) {
```

Comment:
Could you add the `! is_customize_preview()` condition to give access to the features if users visit the Customizer directly? 🙂  (c.f., https://github.com/Automattic/jetpack/pull/36146)

```suggestion
	if ( ! $is_wpcom && wp_is_block_theme() && ! is_customize_preview() ) {
```

Reply:
Thank you! Committed 😁